draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -790,7 +794,7 @@ other flows with that subscriber are denied attempts to prioritize
 themselves. However, the network cannot identify whether the
 prioritized flow is legitimate or malicious.
 
-Requirements: REQ-CLIENT-VALIDATION.
+Such mechanism would satisfy REQ-CLIENT-VALIDATION.
```

Comment:
because we are not prioritizing between flows, I believe we are immune from abuse.  Let's say a flow was all VoIP without any silence detection or anything, for example, so it is a solid stream of bits at a fixed rate.  Every packet could be marked High Priority.  Or Low Priority.  It doesn't matter.  If, while that VoIP is going on, the user adds a file transfer or the user adds screen sharing, then the priority begins to matter -- assuming the client considers the VoIP most important, the VoIP should get the highest priority among the 3 streams on that same flow (file transfer, VoIP, screen share).  A malicious sender would be noticed by the client both in the poorer user experience and in the packets being received which have incorrect priority markings.

Reply:
I agree Dan. Added text to reflect this into security considerations and removed Abuse section.